src/lib.rs

```diff
@@ -264,7 +271,7 @@ impl Plugin for BigBrainPlugin {
 
 /// [`BigBrainPlugin`] system sets. Use these to schedule your own
 /// actions/scorers/etc.
-#[derive(Clone, Debug, Hash, Eq, PartialEq, SystemSet, Reflect)]
+#[derive(Clone, Debug, PartialEq, Eq, Hash, SystemSet, Reflect)]
```

Comment:
[`ScheduleLabel`](https://github.com/bevyengine/bevy/blob/aefe1f0739bb395d2454e05a1a812acab767892c/crates/bevy_ecs/macros/src/lib.rs#L460) may be a good fit here as it's used in [`main_schedule.rs`](https://github.com/bevyengine/bevy/blob/aefe1f0739bb395d2454e05a1a812acab767892c/crates/bevy_app/src/main_schedule.rs#L26), but I didn't see a proper way to utilize it